### ISSUE TITLE
Update stable version in repo from 1.0.2 to 1.2.0

### ISSUE THREAD

**Issue by GermanBluefox** [iobroker-community-adapters/ioBroker.xs1#92]
# Think about update stable version to 1.2.0
**Version**: stable=**1.0.2** (2473 days old) => latest=**1.2.0** (15 days old)
**Installs**: stable=**35** (45.45%), latest=**4** (5.19%), total=**77**

Click to use [developer portal](https://www.iobroker.dev/adapter/xs1/releases)
Click to [edit](https://github.com/ioBroker/ioBroker.repositories/edit/master/sources-dist-stable.json#L3161)

**Do not close this issue manually as a new issue will be created if condition for update still exists.**

Please drop a comment if any reason exists which blocks updating to version 1.2.0 at this time.

Note: This is an automatically generated message and not personally authored by bluefox!
      @mcm1957 for evidence

**Comment by mcm1957** [comment]
on hold
fix of #19 should be done first
